src/views/BuyMYC/BuyMYC.js

```diff
@@ -210,43 +210,43 @@ export default function BuyGMX() {
         )}
 
         {chainId === ARBITRUM && (
-          <div className="BuyGMXGLP-panel">
+          <div className="BuyMYCMLP-panel">
             <div className="buy-card">
               <div className="section-title-content">
-                <div className="card-title">Buy GMX</div>
+                <div className="card-title">Buy MYC</div>
               </div>
               <div className="App-card no-height">
                 <div className="App-card-content no-title">
-                  <div className="BuyGMXGLP-description better-rates-description">
+                  <div className="BuyMYCMLP-description better-rates-description">
                     After you have ETH, set your network to{" "}
                     <a href="https://arbitrum.io/bridge-tutorial/" target="_blank" rel="noopener noreferrer">
                       Arbitrum
                     </a>{" "}
                     then click the button below:
                   </div>
-                  <div className="buy-gmx">
+                  <div className="buy-myc">
                     <Button
                       size="xl"
-                      imgSrc={gmxArbitrum}
+                      imgSrc={mycArbitrum}
                       href="https://app.uniswap.org/#/swap?inputCurrency=ETH&outputCurrency=0xfc5A1A6EB076a2C7aD06eD22C90d7E710E35ad0a"
```

Comment:
This also links to buying the GMX token on Uniswap

Reply:
We dont have an MYC token on uniswap just yet, will leave for now